### PR TITLE
Make Classes Final

### PR DESCRIPTION
**Pull request by kunicmarko20**
I am targeting this branch, because this is a BC break.

### CODE REVIEW

File: src/DependencyInjection/SonataFormatterExtension.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class SonataFormatterExtension extends Extension
+final class SonataFormatterExtension extends Extension
```
Comment (greg0ire) [inline src/DependencyInjection/SonataFormatterExtension.php]: I think the `createEnvironment` method could be made private

**Comment by kunicmarko20** [comment]
Is upgrade note ok?

File: src/Twig/SecurityPolicyContainerAware.php [hunk 2]
```diff
@@ -28,42 +28,42 @@
...
  * @author thomas.rabaix@sonata-project.org
  */
-class SecurityPolicyContainerAware implements SecurityPolicyInterface
+final class SecurityPolicyContainerAware implements SecurityPolicyInterface
 {
     /**
      * @var string[]
      */
-    protected $allowedTags;
+    private $allowedTags;
```
Comment (OskarStark) [inline src/Twig/SecurityPolicyContainerAware.php]: imo you should name the protected -> private move in the `UPGRADE.md`, too

Comment (kunicmarko20) [reply src/Twig/SecurityPolicyContainerAware.php]: well, the class is final so they shouldn't really care?

**Comment by greg0ire** [comment]
You should probably rebase, in order to get rid of merge commits

**Comment by kunicmarko20** [comment]
Merge commits are all mine commits after conflicts because I used github editor, you can squash and merge and we will get only 1 commit on master

**Comment by greg0ire** [comment]
Can't merge actuall, it says: This branch cannot be rebased due to conflicts

**Comment by kunicmarko20** [comment]
I did a rebase locally and I had conflicts but the commit was empty at the end :man_shrugging: , will avoid github editor in future :smile:

**Comment by greg0ire** [comment]
Thanks @kunicmarko20 !

**Pull request merged**